index.js

```diff
@@ -7,24 +7,25 @@ module.exports = function Rivet(url, options) {
   const retryCount = options.retryCount || 0
   delete options.retryCount
```

Comment:
Tests were breaking when I had this inside the new `configureClient` function - that's likely expected, I just don't know enough about this code yet